### DISCUSSION TITLE
Would Frigate be interested in a community supported/maintained list of IP Cameras and related technologies?

### DISCUSSION THREAD

**Discussion by rowskilled14** [blakeblackshear/frigate#15507] [Ask A Question]
The idea is pretty simple, there's a desire for a list of IP Cameras in a wiki-like fashion.  Certain cameras have certain known limitations and issues, while others work great and have special features.  These notable points can all be documented in a single location for the benefit of all.

There's a discussion here which prompted this idea: https://github.com/AlexxIT/go2rtc/issues/1493

___

I've recently started outlining a repo for this task, and will be happy to maintain this project.  The maintainer of go2rtc is on-board with the concept, and will provide what I'm calling, "official unofficial" support for it.  [I defined this term for clarity here](https://github.com/AlexxIT/go2rtc/issues/1493#issuecomment-2515391038).  To briefly reiterate, I use that term to mean that go2rtc makes no promises, guarantees, claims, or statements towards this new project, and is 100% distinct.  Go2rtc isn't responsible for this new project/repo.  However, they _can_ act as a friendly partner, pointing people towards it via a mention in the readme and references in issues/discussions when appropriate to help grow the new project/repo.  When they spot a new camera, they can make a mention of it, create an issue for it, or simply share the idea of contributing with the individual.  No hard obligations or formal commitments are expected - just a friendly community effort.

Given how go2rtc and Frigate are closely related in both nature and camera compatibility, it seems natural to me that I extend my humble request of, "official unofficial" support (as formally defined in the above link) for this idea to the Frigate team as well.  Frigate is a great NVR solution, and I'm hoping to contribute towards it both with this project and another which I am still fine tuning.  I think this has a great potential to expand Frigate's existing documentation on camera specific issues, and would benefit quite a lot of people.  It wouldn't require much effort on the part of the Frigate team - just a mention where appropriate as already discussed.  Arguably, this could serve to *reduce* the effort you take in interacting with support for some types of support requests, but I won't speak for you on that topic.

For these reasons, I'm hoping that Frigate can get onboard with this idea, and extend, "official unofficial" support.  I'm open to all criticism and healthy discussion on the topic.  My goal here is to help everyone out - go2rtc, Frigate, the average end user, and the power user.  I'm looking forward to your thoughts and ideas.  

Thank you very much!

**Comment by NickM-27** [comment]
I think many users would find use for this. I'd suggest calling it "community supported" list of supported cameras, and it could be linked to from the recommended hardware section in the docs.

**Comment by hawkeye217** [comment]
I agree with @NickM-27, it certainly would be helpful. "Community supported" is a term we already use in our docs, so Frigate users would be familiar with that in terms of what (and what not) to expect out of the official Frigate maintainers.

**Reply by rowskilled14** [reply]
That works great for me!  The reason I came up with that term is because if possible, I would like active encouragement from the Frigate team as well.  Docs is 100% great.  The active encouragement would come into play for when you see a new camera pop up into issues and discussions.  It seems like a natural progression which would benefit everyone as described before, so I don't think anyone would have issues with that, and so I just wanted to emphasize it upfront for transparency.

I'll move forward with community supported as the desired term from here.  I'll update you both here once I have the formatting and structure of the repo down - I'm still muddling over that a touch.  I think the Github Wiki feature is the right choice, since it allows for the easiest community editing.  Open to suggestions.

**Reply by rowskilled14** [reply]
### Just an update:

The past two months have been quite busy for me, even given the time of year.  I've made some progress, and will try to pick up the pace soon.

I've made some more progress with the issue templating and I have a basic Github Actions workflow in place.  The Github Actions workflow is intended to make adding entries manually to the wiki very easy after users fill out the given template.  This was a design choice to try to cut back on entries which don't follow existing formatting practices, similar to traditional issues/PRs for git style systems.  The idea was for curation of entries in a very easy and nearly hands free approach rather than having a fully open wiki which is prone to style differences between contributors.  I may revisit options here in the future, but I think it'd be a great starting point.

I am also working on a generic, "Intro to IP cameras" type of guide to throw into the Wiki since it seems extremely relevant and potentially helpful.  I still have a fair bit I want to do with that, however.  Long term I'd like it to be fairly all inclusive, with better sources.  As of now, it's mostly just me going from memory and experience, and in a perfect world it should read a bit more objective than the current WIP.  Once I get this, "Intro to IP cameras" guide/wiki filled in a bit more, there's only a template or two more that I need to fill out before I'm ready to move forward on my end of things.  

Feedback and thoughts are of course welcome.

**Comment by ElectronicBattle** [comment]
I'd certainly like this resource to exist; I can contribute my cameras and experiences to it, especially if the format of the wiki helps the person submitting an entry to fill in sufficiently detailed information so as to make it useful (a bit like submitting a bug report here forces you to enter the ```config.yaml``` etc., so could the wiki require an entry from ```ffprobe``` for example).

**Comment by primevisionservices** [comment]
list of cameras that the community used and worked well, features such as two-way audio, could be interesting
